roles/edpm_ovn_dpdk/meta/argument_specs.yml

```diff
@@ -0,0 +1,66 @@
+---
+argument_specs:
+  # ./roles/edpm_ovn_dpdk/tasks/main.yml entry point
+  main:
+    short_description: The main entry point for the edpm_ovn_dpdk role.
+    options:
+      edpm_ovn_dpdk_pmd_core_list:
+        type: str
+        default: ""
+        description: OVN DPDK PMD Core list
+      edpm_ovn_dpdk_lcore_list:
+        type: str
+        default: ""
+        description: OVN DPDK LCore list
+      edpm_ovn_dpdk_memory_channels:
+        type: int
+        default: ""
+        description: OVN DPDK memory channels
+      edpm_ovn_dpdk_extra:
+        type: str
+        default: ""
+        description: OVN DPDK extra configs
+      edpm_ovn_dpdk_socket_memory:
+        type: str
+        default: ""
+        description: OVN DPDK Socket Memory
+      edpm_ovn_dpdk_revalidator_cores:
+        type: str
+        default: ""
+        description: DPDK revalidator core list
+      edpm_ovn_dpdk_handler_cores:
+        type: str
+        default: ""
+        description: OVN DPDK handler Core list
+      edpm_ovn_dpdk_emc_insertion_probablity:
+        type: str
+        default: ""
+        description: OVN DPDK EMAC insertion probablity
+      edpm_ovn_dpdk_enable_tso:
+        type: bool
+        default: false
+        description: OVN DPDK TSO enable or not
+      edpm_ovn_dpdk_pmd_auto_lb:
+        type: bool
+        default: false
+        description: OVN DPDK PMD auto load balance enable or not
+      edpm_ovn_dpdk_pmd_load_threshold:
+        type: str
+        default: ""
+        description: OVN DPDK PMD load threshold
+      edpm_ovn_dpdk_pmd_improvement_threshold:
```

Comment:
podified-multinode-edpm-deployment-crc job is reporting issue somewhere here. See https://logserver.rdoproject.org/28/328/3d51725ccb2b3cfcc85f0e37d773d8552e69d7b9/github-check/podified-multinode-edpm-deployment-crc/19348b1/controller/ci-framework-data/artifacts/must-gather.local.1599254522418717781/quay-io-openstack-k8s-operators-openstack-must-gather-sha256-070a98749455b097c66a1a25ff827a3c0ee7385f1f0c8bf54f5b15438501f2bd/namespaces/openstack/pods/dataplane-deployment-configure-network-openstack-edpm-84q24/logs/dataplane-deployment-configure-network-openstack-edpm-84q24.log